src/mSupplyMobileApp.js

```diff
@@ -143,7 +143,8 @@ class MSupplyMobileAppContainer extends React.Component {
     await new Promise(resolve => {
       this.setState({ isLoading: true }, () => setTimeout(resolve, 1));
     });
-    functionToRun();
+
+    await functionToRun();
```

Comment:
awaiting a non-promise converts it to a promise, so this is OK, apparently!

Reply:
If it works, it works!